src/entity/Action.ts

```diff
@@ -0,0 +1,32 @@
+import { Session } from './Session';
+import { Command } from './Command';
+import { Exception } from './Exception';
+
+import {
+	Column,
+	Entity,
+	ManyToOne,
+	OneToMany,
+	PrimaryGeneratedColumn,
+} from 'typeorm';
+
+@Entity()
+export class Action {
```

Comment:
Maybe we should rename this to Event since not all actions (such as exceptions) are actual actions